source/sync/migrate-sync-modes.txt

```diff
@@ -27,9 +27,8 @@ Partition-Based Sync.
 Requirements
 ------------
 
-Currently, the Swift, Kotlin, and Java client SDKs support migrating your App's
-backend from Partition-Based Sync to Flexible Sync. The .NET, Node.js, and
-React Native SDKs do not support backend Sync mode migration yet.
+The .NET SDK does not support backend Sync mode migration yet. This feature does
+not apply to clients that use the C++, Flutter, or Web SDKs.
```

Comment:
Should we mention _why_ this feature isn't available in those SDKs? (i.e. they have never supported Partition-Based Sync?)

I just wonder if people will see them called out here and wonder if there's a plan to add them or otherwise wonder why they're called out.

Reply:
I initially added that detail in, but felt like it was too much info for what's effectively an admonition in the "Requirements" section. Looking at it now, it may be best to not mention them at all. 🤔